### ISSUE TITLE
--exclude-table-in-database requires an arg

### ISSUE THREAD

**Issue by jfrost** [pgexperts/flexible-freeze#25]
Looks like `--exclude-table-in-database` requires an argument or the script blows up:

Doesn't work:
```/usr/bin/python2.7 /usr/local/bin/flexible_freeze/scripts/flexible_freeze.py --minutes 1 -H localhost
Traceback (most recent call last):
  File "/usr/local/bin/flexible_freeze/scripts/flexible_freeze.py", line 134, in <module>
    for elem in args.exclude_table_in_database:
TypeError: 'NoneType' object is not iterable
```

Works fine:
```
/usr/bin/python2.7 /usr/local/bin/flexible_freeze/scripts/flexible_freeze.py --minutes 1 -H localhost --exclude-table-in-database this_is_not_a_real_database.this_is_not_a_real_table
```

**Comment by qdw** [comment]
With the latest version (3e6a60d), rather than blowing up in the middle of the program, you get the following error message at the beginning:

```
$ python scripts/flexible_freeze.py --minutes 1 -H localhost --exclude-table-in-database
usage: flexible_freeze.py [-h] [-m RUN_MIN] [-d DBLIST] [-T TABLES_TO_EXCLUDE]
                          [--exclude-table-in-database EXCLUDE_TABLE_IN_DATABASE]
                          [--vacuum] [--pause PAUSE_TIME]
                          [--freezeage FREEZEAGE] [--costdelay COSTDELAY]
                          [--costlimit COSTLIMIT] [-t] [--enforce-time]
                          [-l LOGFILE] [-v] [--debug] [-U DBUSER] [-H DBHOST]
                          [-p DBPORT] [-w DBPASS]
flexible_freeze.py: error: argument --exclude-table-in-database: expected one argument
```

This seems like the right behavior to me. Do you have a use case for passing a possibly-empty list of tables to exclude?

**Comment by jfrost** [comment]
Well, my use case was not specifying the argument at all. i.e. consider all tables.

**Comment by qdw** [comment]
Ah, I see. Should be fixed in 2ec7b7b (just pushed).
